db/migrate/20220512183030_add_status_to_articles.rb

```diff
@@ -0,0 +1,5 @@
+class AddStatusToArticles < ActiveRecord::Migration[7.0]
+  def change
```

Comment:
Nbd, but you could theoretically have done this in the one migration. But it doesn't matter